webSites/webSites-war/src/main/java/org/silverpeas/components/websites/control/WebSiteSessionController.java

```diff
@@ -634,10 +635,12 @@ private synchronized void delDir(File dir) {
           delDir(dirFile);
         }
       }
-      dir.delete();
+      if (!dir.delete()) {
+        SilverLogger.getLogger(this).warn("Cannot delete directory {0}", dir.getPath());
+      }
     } catch (Exception e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1166)